src/SonarQube.Bootstrapper/SonarQube.Bootstrapper.csproj

```diff
@@ -18,4 +18,15 @@
     <ProjectReference Include="..\SonarQube.TeamBuild.PreProcessor\SonarQube.TeamBuild.PreProcessor.csproj" />
     <ProjectReference Include="..\SonarScanner.Shim\SonarScanner.Shim.csproj" />
   </ItemGroup>
-</Project>
+  <!-- Conditionally reference project only if the framework is classic .NET Framework (not core or standard). -->
+  <ItemGroup Condition="('$(TargetFrameworkIdentifier)' == '.NETFramework')">
+      <ProjectReference Include="..\SonarQube.TeamBuild.Integration.Legacy\SonarQube.TeamBuild.Integration.Legacy.csproj" />
+  </ItemGroup>
+  <!-- Define a constant if the framework is classic .NET Framework. -->
+  <Target Name="AddNetFrameworkMacro" BeforeTargets="CoreCompile" Condition="('$(TargetFrameworkIdentifier)' == '.NETFramework')">
```

Comment:
I couldn't find a way to reuse the expression in "Condition" here and few lines above.